### DISCUSSION TITLE
Reindex search when repository is renamed and another repository uses that name

### DISCUSSION THREAD

**Discussion by 3UR** [community/community#67155] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

If I have a repository -> **repo1** and the I search for something it becomes indexed but then if I was to rename **repo1** -> **repository123** and then create a brand new repository under the name **repo1** the search will be stuck to the old results and wont reindex

**Comment by look** [comment, marked as answer]
Sorry for the problem. Is this still affecting you? The search index does not update immediately after changes like this, but it should reflect the changes after some time.

**Reply by 3UR** [reply]
sorry for the late reply, yes it isnt happening anymore thanks
